core/src/main/java/com/hcl/ecomm/core/servlets/CountryStateServlet.java

```diff
@@ -0,0 +1,95 @@
+package com.hcl.ecomm.core.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.hcl.ecomm.core.services.CartService;
+import com.hcl.ecomm.core.services.ShippingInfoService;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.servlets.HttpConstants;
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+@Component(service = Servlet.class, property = { "sling.servlet.paths=/bin/hclecomm/countrystatelist",
+		"sling.servlet.method=" + HttpConstants.METHOD_GET, "sling.servlet.extensions=json" })
+public class CountryStateServlet extends SlingSafeMethodsServlet {
+
+	private static final long serialVersionUID = 4016057296495129474L;
+	private static final Logger LOG = LoggerFactory.getLogger(CountryStateServlet.class);
+
+	@Reference
+	private ShippingInfoService shippingInfoService;
+
+	protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse response)
+			throws ServletException, IOException {
+
+		LOG.info("inside CountryState Servlet  doGET method");
```

Comment:
@nitheshkumar-hcl : remove log.info

Reply:
Committed the changes as per the review comments

Reply:
Committed the changes as per the review comments